### ISSUE TITLE
didn't converge with nuscene dataset

### ISSUE THREAD

**Issue by nunuisacat** [tarashakhurana/emergent-occ-forecasting#4]
Hello dear developers,

I tried to train the model using nuscenes v1.0 mini dataset and the default hyperparameter sets except for the voxel size, which I changed it to 0.5.  And with the model obj_guided.

Unfortunately the training didn't converge at all.
I have also changed the learning rate ranging from 1e-1 to 5e-3, but didn't improve anything.

Do you have any idea what is going wrong?
